src/libs/dataUseTranslation.js

```diff
@@ -162,12 +162,13 @@ export const consentTranslations = {
   }
 };
 
-const getOntologyName = async(obolibraryURL) => {
-  const ontology = await searchOntology(obolibraryURL);
-  return ontology.label;
+const getOntologyName = async(urls) => {
+  const joinedUrls = urls.join(',');
+  const ontology = await searchOntology(joinedUrls);
+  return ontology.map(data => data.label);
 };
```

Comment:
ToL: the new API can accept a shortened version with just the DOIDs which might be something to consider for an optimization. For some of our current insane dev DARs, we have hundreds of ontologies which can translate to thousands of characters if we use the full url (see https://stackoverflow.com/questions/812925/what-is-the-maximum-possible-length-of-a-query-string#:~:text=Although%20officially%20there%20is%20no,a%20max%20of%202048%20characters.)

Reply:
On the Dataset side, there are no problematic cases where the urls would exceed 1024 characters so the dataset registration page is not affected. 
On the DAR side, we'll start hitting problems with ~26 or more datasets (1024/39). The Dar Collection Review page uses this code so there is a risk of breakage here with large DAR Collections.